app/src/main/java/org/p2p/wallet/utils/TransactionImageViewExtensions.kt

```diff
@@ -0,0 +1,15 @@
+package org.p2p.wallet.utils
+
+import org.p2p.uikit.R
+import org.p2p.uikit.atoms.UiKitTransactionImageView
+import org.p2p.wallet.transaction.model.TransactionStatus
+
+fun UiKitTransactionImageView.setStatus(status: TransactionStatus?) {
```

Comment:
why cant we put it as a method?

Reply:
why method ? =(
UiKitTransactionImageView is in UiKit
TransactionStatus is an App model
thats why i done this stuff

Reply:
i see, thanks